src/sentry/data_export/models.py

```diff
@@ -121,6 +120,16 @@ def email_failure(self, message):
         msg.send_async([self.user.email])
         self.delete()
 
+    def _get_file(self):
```

Comment:
We could potentially just make this
```
@property 
def file(self):
```

But if you prefer the explicit function call I'm ok with that.

Reply:
I'd prefer to not make it look like a model field.